src/apps/menu/menu.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { FC, useEffect, useState } from "react";
+import profileIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/basic/icon-profile.svg";
+import CloseIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/CloseLarge.svg";
+import { Link } from "../../components/atoms/link";
+import MenuNavigationList from "./menu-navigation-list/menu-navigation-list";
+import MenuNotification from "./menu-notification/menu-notification";
+import { useConfig } from "../../core/utils/config";
+import { useText } from "../../core/utils/text";
+import ArrowIcon from "../../components/atoms/icons/arrow/arrow-white";
+import {
+  useGetPatronInformationByPatronIdV2,
+  useGetLoansV2,
+  useGetReservationsV2,
+  useGetFeesV2
+} from "../../core/fbs/fbs";
+import { AuthenticatedPatronV6 } from "../../core/fbs/model/authenticatedPatronV6";
+import { useGetV1UserLoans } from "../../core/publizon/publizon";
+import {
+  mapPublizonLoanToLoanType,
+  mapFBSLoanToLoanType
+} from "../../core/utils/helpers/list-mapper";
+import { LoanType } from "../../core/utils/types/loan-type";
+import {
+  filterLoansOverdue,
+  filterLoansSoonOverdue,
+  getReadyForPickup
+} from "../../core/utils/helpers/general";
+import { useUrls } from "../../core/utils/url";
+
+interface MenuNavigationDataType {
+  name: string;
+  link: string;
+  dataId: string;
+}
+
+const Menu: FC = () => {
+  const t = useText();
+  const config = useConfig();
+  const { data: patronData } = useGetPatronInformationByPatronIdV2();
+  const { data: patronReservations } = useGetReservationsV2();
+  const { data: publizonData } = useGetV1UserLoans();
+  const { data: fbsData } = useGetLoansV2();
+  const { data: fbsFees } = useGetFeesV2();
```

Comment:
I think the cache will save us in this case. No data is updated from this view, so invalidation is not relevant.
I added loading state as a TODO, as we talked about this might be a global decision we have to make at some point.